examples/minimal-appdir/README.md

```diff
@@ -0,0 +1,16 @@
+# Minimal Next.js App Router example for UploadThing
+
+[![stackblitz-logo](https://res.cloudinary.com/practicaldev/image/fetch/s--jByI8PG5--/c_fill,f_auto,fl_progressive,h_320,q_auto,w_320/https://dev-to-uploads.s3.amazonaws.com/uploads/organization/profile_image/5299/3d6e2f74-98b6-437d-99d9-8222bb7ffad9.png) Open in Stackblitz](https://stackblitz.com/github/pingdotgg/uploadthing/tree/main/examples/minimal-appdir)
```

Comment:
replace `tree/main` with `tree/examples-revamp` if you wanna try the link now

Reply:
This was a good idea.